src/snapred/backend/service/LiteDataService.py

```diff
@@ -28,6 +33,7 @@ def reduceLiteData(
         self,
         inputWorkspace: WorkspaceName,
         outputWorkspace: WorkspaceName,
+        runNumber: str,
```

Comment:
Is there a way to remove this input?  It should be present in the workspace name.

Reply:
Yep, updated to just grab from workspace name